### PR TITLE
DO-1850 Store unfinished construction rights

### PR DESCRIPTION
**Pull request by OlexandrTopuzov**

### CODE REVIEW

File: business_register/converter/declaration.py [hunk 1]
```diff
@@ -733,6 +733,44 @@ def save_luxury_item(self, luxuries_data, declaration):
...
+            first_name = data.get('ua_firstname')
+            middle_name = data.get('ua_middlename')
+            if (
+                    last_name not in self.NO_DATA
+                    or first_name not in self.NO_DATA
+                    or middle_name not in self.NO_DATA
+            ):
+                full_name = f'{last_name} {first_name} {middle_name}'
+            else:
+                full_name = ''
```
Comment (roman-tiukh) [inline business_register/converter/declaration.py:764]: first_name and last_name is required

File: business_register/converter/declaration.py [hunk 2]
```diff
@@ -733,6 +733,44 @@ def save_luxury_item(self, luxuries_data, declaration):
...
+            PropertyRight.objects.create(
+                property=unfinished_construction_property,
+                type=type,
+                additional_info=additional_info,
+                share=share,
+                pep=pep,
+                full_name=full_name,
```
Comment (roman-tiukh) [inline business_register/converter/declaration.py:771]: do we store first, last, middle names separately? I think we should do this

File: business_register/converter/declaration.py [hunk 3]
```diff
@@ -733,6 +733,44 @@ def save_luxury_item(self, luxuries_data, declaration):
...
+                share = None
+            owner_info = data.get('rightBelongs')
+            pep = None
+            # TODO: store value from ENIGMA
+            if owner_info not in self.NO_DATA and owner_info not in self.ENIGMA:
+                pep = Pep.objects.filter(nacp_id=int(owner_info)).first()
+            other_owner_info = data.get('rights_id')
+            # Store value 'Інша особа (фізична або юридична)'
+            if not pep and other_owner_info and other_owner_info not in self.ENIGMA:
+                pep = Pep.objects.filter(nacp_id=int(other_owner_info)).first()
```
Comment (roman-tiukh) [inline business_register/converter/declaration.py:752]: wait for Emma`s PR. nacp_id will be a array of ids

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/converter/declaration.py [hunk 4]
```diff
@@ -733,6 +733,44 @@ def save_luxury_item(self, luxuries_data, declaration):
...
+            share = data.get('percent-ownership')
+            if share not in self.NO_DATA:
+                share = float(share.replace(',', '.'))
+            else:
+                share = None
+            owner_info = data.get('rightBelongs')
+            pep = None
+            # TODO: store value from ENIGMA
+            if owner_info not in self.NO_DATA and owner_info not in self.ENIGMA:
+                pep = Pep.objects.filter(nacp_id=int(owner_info)).first()
```
Comment (EmmaSamoilova) [inline business_register/converter/declaration.py:748]: pep = self.find_person(owner_info)

File: business_register/converter/declaration.py (hunk 3 above)
Comment (EmmaSamoilova) [inline business_register/converter/declaration.py:752]: pep = self.find_person(other_owner_info)

**Pull request closed without merging**